yoga/Yoga.cpp

```diff
@@ -3527,9 +3561,8 @@ static void YGNodelayoutImpl(
   }
 }
 
-uint32_t gDepth = 0;
-bool gPrintChanges = false;
-bool gPrintSkips = false;
+#define PRINT_CHANGES false
+#define PRINT_SKIPS false
```

Comment:
This is probably OK, but it is safer to use `#define XXX 0` here for safer compatibility against various pre-processors, that look for a "C expression of integer type" (gcc) or "must have integral type and can include only integer constants, character constants, and the defined operator" (msvc).